spec/classes/server_spec.rb

```diff
@@ -367,6 +367,20 @@
         it { is_expected.to contain_file('/etc/zabbix/zabbix_server.conf').without_content %r{^StartODBCPollers=1} }
       end
 
+      context 'with zabbix_server.conf and version 7.0' do
+        next if facts[:os]['name'] == 'Debian' && facts[:os]['release']['major'].to_i <= 11
```

Comment:
shouldn't the check be in the template iteself? If the parameter is set for an older zabbix version, the tests will pass but it will fail in production? templates/zabbix_server.conf.erb line 968 also has a version check.